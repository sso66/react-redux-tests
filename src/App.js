import React from 'react';
import logo from './logo.svg';
import './App.sass';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          TDD/BDD: <code>React-Redux Tests</code>
        </p>
      </header>
    </div>
  );
}

export default App;
