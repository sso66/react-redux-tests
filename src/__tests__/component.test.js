import React from 'react';
import App from '../App';
import { create } from 'react-test-renderer'

describe('actions', ()=> {

    it('testing app button', () => {
        let tree = create(<App />)
        expect(tree.toJSON()).toMatchSnapshot();
    });
        
    // it('should create an action to add a todo', () => {
    //     const expected = { type: 'ADD_RECIPE', payload: 'test' }

    //     expected(actions.addRecipe('test')).toEqual(expected);
    // });
});