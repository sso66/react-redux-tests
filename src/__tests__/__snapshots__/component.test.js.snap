// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions testing app button 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      TDD/BDD: 
      <code>
        React-Redux Tests
      </code>
    </p>
  </header>
</div>
`;
